src/main/java/com/redhat/cloud/custompolicies/app/VerifyEngine.java

```diff
@@ -45,10 +48,36 @@
    * @return A message with verification/store results.
    */
   @POST
+  @Path("/trigger")
   @Consumes("application/json")
   @Produces("application/json")
   Msg store(FullTrigger trigger,
             @QueryParam("dry") boolean isDryRun,
             @HeaderParam("Hawkular-Tenant" ) String customerId
   );
+
+  /**
+   * Update a trigger
+   * @param trigger Policy wrapped in a data structure, the engine expects
+   * @param customerId Id of the customer this policy belongs to.
+   * @return A message with update results
+   */
+  @PUT
+  @Consumes("application/json")
+  @Produces("application/json")
+  @Path("/trigger/{triggerId}")
+  Msg update(@PathParam("triggerId") String triggerId,
+             FullTrigger trigger,
+             @HeaderParam("Hawkular-Tenant") String customerId);
+
+  /**
+   * Delete the trigger with the given id
+   * @param triggerId Id of the trigger to delete
+   * @param customerId Id of the customer this policy belongs to.
+   */
+  @DELETE
+  @Consumes("application/json")
+  @Path("/{triggerId}")
```

Comment:
It doesn't seem consistent with POST and PUT, is it expected ?  I would have expected /trigger/{triggerId}

Reply:
It is correct, it's actually referencing to the same as PUT/POST to ``/{triggerId}`` would. The ``trigger/{triggerId}`` is full trigger creation, while ``{triggerId}`` would point to a trigger. Full trigger includes definitions for conditions, dampenings etc, but the trigger itself is only the trigger (yet, destroying a trigger makes sure the rest of the underlying definitions are removed also - since they're orphan).

Reply:
That sounds complicated but outside this PR. Feel free to merge @burmanm